test-sbt/jvm/src/main/scala/zio/test/sbt/ZTestRunnerJVM.scala

```diff
@@ -44,7 +42,7 @@ final class ZTestRunnerJVM(val args: Array[String], val remoteArgs: Array[String
   )
 
   def done(): String = {
-    shutdownHook()
```

Comment:
So then it should be 
```scala
 def done(): String = 
    shutdownHook.fold("") { shutdownHook => 
        shutdownHook()
        ....
   }
```
because first instance will report false empty result - `No tests were executed` instead of valid results, shutdown the ZIO runtime and second instance will no print results at all. Just tested this behaviour on my reproducible repo.

Reply:
Absolutely right, thanks for the correction!